src/lib.rs

```diff
@@ -420,6 +425,59 @@ fn to_valid_pkg_name(project_name: &str) -> String {
     }
 }
 
+fn create_dotnet_webassembly_project(project_name: &str, target_dir: &std::path::PathBuf) -> anyhow::Result<()> {
```

Comment:
The template shouldn't be bootstrapped using the dotnet CLI and instead should be added to `templates` directory in this repo same as other templates. The template should replicate this look and behavior 
![image](https://github.com/tauri-apps/create-tauri-app/assets/48618675/44dedcc7-2327-4580-9f94-e1f5195792db)

Reply:
I'm going to take care of appearance and behavior. Indeed, at first I wanted to do as you just told me but I encountered a problem, in C# we have .csproj files that serve as a repository for our project, that said, I couldn't rename the .csproj with the project nameIndeed, at the beginning I wanted to do as you just told me but I encountered a problem, in C# we have .csproj files that serve as a repository for our project, that said, I couldn't rename the .csproj with the project name, y to Is there a way to change the name of a file with the name of the project like this ?
![image](https://github.com/tauri-apps/create-tauri-app/assets/126319968/3413ffeb-c7fd-4302-8727-2c08aa8ea4ed)

Reply:
I've just finished making the template, I'm still having the problem renaming my .csproj file

![image](https://github.com/tauri-apps/create-tauri-app/assets/126319968/461e8faa-8495-4a47-b476-42abdddda3f3)

Reply:
@zenitogr after these lines https://github.com/jimmy-mll/create-tauri-app/blob/96bb2a4477ef099a0c37bcd5b95b87045b3df16d/src/template.rs#L380-L387 you can add `let new file_name = lte::render(file_name, manifest_template_data)?;`

Reply:
thank you, I will do that today

Reply:
That's okay, I'm done everything is done according to your requirements @amrbashir